easybuild/easyconfigs/l/libcroco/libcroco-0.6.13-foss-2020a.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'libcroco'
+version = '0.6.13'
+
+homepage = 'https://github.com/GNOME/libcroco'
+description = """Libcroco is a standalone css2 parsing and manipulation library."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
```

Comment:
Does this really belong to foss level? Seems like a GCCcore would be a better fit as I can't imagine it depends on BLAS or MPI.

Reply:
If I change the toolchain to 'GCCcore', do I update this PR or submit an new one?

Reply:
You can modify this PR. You can use : to remove the old file from the PR, like this `eb --update-pr 12118 --pr-commit-msg="Move to GCCcore" :librsvg-2.50.2-foss-2020a.eb librsvg-2.50.2-GCCcore-10.2.0.eb  (samd thing for libcroco)`